lib/galaxy/config/schemas/config_schema.yml

```diff
@@ -1151,6 +1151,27 @@ mapping:
           Shown in warning box to users that were not activated yet.
           In use only if activation_grace_period is set.
 
+      default_paired_list_filter_name:
+        type: str
```

Comment:
Also struggled a bit here with creating the object that I wanted to create in the config. I ended up just caving and creating three strings and then buidling the Object with an Array in the Vue component, but I'm not sure if that's the best call.

I was unable to create an array here, let alone an object. It looks like everything else in this file is either a string, int, float, or boolean, so there was nothing to use as inspiration. When looking up xml types, it seemed easy enough, but this file doesn't comply with the defintiion of xml types (i.e. str vs. string ). I came across this file `lib/galaxy/config/schema.py` but wasn't sure exactly how to change it to allow arrays or objects.  

All that to say, my current implementation of three strings works fine, but I wondered if it should be more concise.